packages/cli/lib/projects.js

```diff
@@ -48,6 +49,11 @@ const PROJECT_STRINGS = {
   },
 };
 
+const containsNestedProject = async projectPath => {
+  const files = await walk(projectPath);
+  return files.find(file => path.basename(file) === 'hsproject.json');
```

Comment:
we should probably create a const for `hsproject.json` if one doesn't already exist